pkgs/tools/package-management/lix/common.nix

```diff
@@ -184,10 +191,15 @@ stdenv.mkDerivation {
       ''}
     '';
 
-  mesonBuildType = "release";
+  # -O3 seems to anger a gcc bug and provide no performance benefit.
+  # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=114360
+  # We use -O2 upstream https://gerrit.lix.systems/c/lix/+/554
+  mesonBuildType = "debugoptimized";
```

Comment:
hmm, i think we explicitly want to separate the debug stuff in a different output, i don't think debugoptimized does that?

Reply:
separateDebugInfo does that. I'm not sure if our Mac builds are relatedly haunted.